### ISSUE TITLE
Ajouter LICENSE.md et LICENSE.fr.md

### ISSUE THREAD

**Issue by SamGuay** [osumontreal/meta#3]
Même si le README.md mentionne les licenses qui s'appliquent, je crois qu'il pourrait être pertinent  de mettre en exemple les fichiers qui contiennent les licenses ou du moins expliciter ce qu'elles permettent dans le README( ou linker vers [le lien user-friendly](https://creativecommons.org/licenses/by-sa/4.0/deed.fr )? 

Comme ce n'est pas tout le monde qui est familier avec les licences (dont moi :raising_hand_man:), ça permettrait de se retrouver rapidement.

Est-ce qu'il y a des sites qui décrivent les autres licenses analogues au CC licenses?

**Comment by dannycolin** [comment]
> Est-ce qu'il y a des sites qui décrivent les autres licenses analogues au CC licenses?

Oui, mais je laisserais les liens vers les licenses complètes, car on est légalement obligé de les distributer avec nos trucs. Par contre, tu peux, dans un paragraphe juste au-dessous, ajouter des liens pour les néophytes.
